### ISSUE TITLE
Transporter skip some Node 

### ISSUE THREAD

**Issue by gonzo1247** [TrinityCore/TrinityCore#25667]
**Description:**
Some Transporters skip their Nodes if they teleport or reach end. I test it for the Transporter 203626
I add some extra Log Out Messages to see what is happend while the Transport move. 
203623 has from 0 to 120 Nodes in TaxiPathNode.dbc 
At node 119 he has a wait time from 30 seconds.  But if the Transport reach node 119 didn't start the Pause "Move" rather this Node is marked as "IsTeleportFrame" and the Transport Teleport itself to a uncorrect position(under the map) and wait there. After the 30 Seconds, transporter port himself to his first node and begin is cycle.

At other Point: (the Transporter move between Kalimdor and Eastern Kingdoms) the last Node Point in Kalimdor is 25, the first in Eastern Kingdoms is Node 26.
But at reaching Node 24 the teleport begins and continues at Node 27. Skip here 2 Nodes.

**Expected behaviour:**

idk if this on Teleport with skipped notes correct, but the transporter didn't should teleport too early.
Its possible that this affects all other Transports.

**Steps to reproduce the problem:**

Go Ingame and go on a Transporter and drive with him.

**Branch(es):**

Testet on 4.34 but affect certainly master

**Comment by illfated** [comment]
If you are using 4.3.4, I presume you are not using TrinityCore source.

**Comment by Faq** [comment]
You might go report issue on https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/commits/master

**Comment by illfated** [comment]
https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/issues seems like a useful page to start from when opening a ticket.

https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/commits/master is good if you want to check the commit history (changes).

**Comment by gonzo1247** [comment]
I test it now with commit 16b39a448acbe8ace88550a367be8e6bf565b00d and here is the same problem

**Comment by illfated** [comment]
Fair enough. Just need to wait for someone to look into your issue and see if they have any ideas of how to work out a solution.

**Comment by Ovahlord** [comment]
This issue should be valid for all branches. When a transport starts its way to a teleport node, the transport gets teleported immdiately before even reaching the position of that node. That's pretty much all of this issue.
